diff_drive_controller/include/diff_drive_controller/autodiff_integrate_function.h

```diff
@@ -0,0 +1,145 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ *  Copyright (c) 2013, PAL Robotics, S.L.
```

Comment:
@mikepurvis Should I use a Clearpath Robotics copyright for the new files I've created here? If that's the case, could you provide me a link to an example?